test/SequenceOverflow.cpp

```diff
@@ -44,7 +44,7 @@ TEST_F(SequenceOverflow, Block1)
     uint8_t output[ARRAY_LENGTH];
 
     EXPECT_TRUE(serialize_sequence_uint8_t(&writer, input, SEQUENCE_LENGTH_OVERFLOW));
-    EXPECT_FALSE(deserialize_sequence_uint8_t(&reader, output, &output_size, ARRAY_LENGTH));
+    EXPECT_FALSE(deserialize_sequence_uint8_t(&reader, output, ARRAY_LENGTH, &output_size));
```

Comment:
For consistence, change ARRAY_LENGTH with ARRAY_CAPACITY

Reply:
Done
Do it also with SEQUENCE_LENGTH to SEQUENCE_SIZE.